classes/controllers/grid/users/reviewer/PKPReviewerGridHandler.php

```diff
@@ -752,8 +752,9 @@ public function reviewRead($args, $request)
             $newReviewData['quality'] = $newReviewData['dateRated'] = null;
         }
 
-        // if the review assignment had been unconsidered, update the flag.
-        $newReviewData['considered'] = $reviewAssignment->getConsidered() === ReviewAssignment::REVIEW_ASSIGNMENT_NEW
+        // if the review assignment had been unconsidered or only viewed but not considered, update the flag.
+        $newReviewData['considered'] = ($reviewAssignment->getConsidered() === ReviewAssignment::REVIEW_ASSIGNMENT_NEW || 
+                                       $reviewAssignment->getConsidered() === ReviewAssignment::REVIEW_ASSIGNMENT_VIEWED)
             ? ReviewAssignment::REVIEW_ASSIGNMENT_CONSIDERED
```

Comment:
Somewhat related problem while testing. I noticed that I was getting `REVIEW_ASSIGNMENT_RECONSIDERED`, even after simple confirmation. Thats because we have also the REVIEW_ASSIGNMENT_VIEWED now.